api/src/main/java/org/pmiops/workbench/absorb/AbsorbServiceImpl.java

```diff
@@ -39,6 +41,9 @@ public Boolean userHasLoggedIntoAbsorb(Credentials credentials) {
 
   @Override
   public List<Enrollment> getActiveEnrollmentsForUser(Credentials credentials) throws ApiException {
+    log.info(
+        String.format("Fetching Absorb enrollments for Absorb user id `%s`", credentials.userId));
```

Comment:
I added logging before and after sections we call Absorb APIs. This should allow us to identify unexpected Absorb API responses and manually reproduce errors.